### ISSUE TITLE
Why doesn't the device have a name

### ISSUE THREAD

**Issue by ATM006** [sitewhere/sitewhere#974]
I am puzzled why there is no device name in the device model in the overall design of sitewhere? What's the point of this?
![image](https://user-images.githubusercontent.com/24607498/121691463-99be8700-caf9-11eb-8804-16298ec42180.png)

**Comment by ATM006** [comment]
+

**Comment by derekadams** [comment]
Yes, it probably makes sense to have a `name` field. In the short term, you could use the existing `comments` field or a metadata entry to store the data. We will add a `name` field to the device schema and API in an upcoming release.
